docs/direct-access-urls.md

```diff
@@ -142,16 +154,16 @@ The following endpoints can be used to obtain direct access URLs:
 * `/nodes/{nodeId}/versions/{versionId}/request-direct-access-url`
 * `/deleted-nodes/{nodeId}/request-direct-access-url`
 * `/deleted-nodes/{nodeId}/renditions/{renditionId}/request-direct-access-url`
-  
+
 **Method:** **`POST`**
-  
+
 **Response:** Link to the resource wrapped in a JSON Object which also contains an attachment flag and the DAU expiration date.
-  
+
 **Error Codes:**
-* If there’s no Direct Access URL provider (e.g. Alfresco S3 Connector extension) installed in
+* If there’s no Direct Access URL provider (e.g. Alfresco S3 Connector / Azure Connector extension) installed in
```

Comment:
Trivial: s/extension/module ?

Reply:
Reworded to module - hope that's what you meant.